src/components/05_pages/NodeForm/index.js

```diff
@@ -15,8 +20,17 @@ schemaType = PropTypes.shape({
   properties: PropTypes.objectOf(lazyFunction(lazySchemaType)),
 }).isRequired;
 
+const styles = () => ({
+  container: {
+    display: 'flex',
```

Comment:
There is a little bug hiding here 

it is similar to #202 - single quotes surrounding text properties of CSS styles is invalid.
This applies to 'flex', 'wrap' and 'column'

I hope this helps.